scripts/requirements.txt

```diff
@@ -1,4 +1,5 @@
 # zephyr-keep-sorted-start
+devicetree
```

Comment:
Is this not already required by the base Zephyr requirements.txt?

Reply:
No I couldn't find it in any of the `scripts/requirements-*.txt` files

Reply:
Ah, I see it's already added. Yeah, I think you can remove devicetree here